plugins/maven-crawler/src/main/java/eu/f4sten/mavencrawler/MavenCrawlerArgs.java

```diff
@@ -27,7 +27,6 @@ public class MavenCrawlerArgs {
     @Parameter(names = "--mc.kafkaOut")
     public String kafkaOut = DefaultTopics.INGEST;
 
-    @Parameter(names = "--mc.indexUrl")
-    public String indexUrl = "https://repo1.maven.org/maven2/.index/nexus-maven-repository-index.%d.gz";
-
+    @Parameter(names = "--mc.easyIndexUrl")
+    public String easyIndexUrl;
```

Comment:
Is it possible to the keep Maven URL as the default and also allow the user to override it if needed?

Reply:
You are mixing up two different things! The "Easy Index" is an abstraction that I have created. It exposes the contents of the Maven index files via a convenient REST API. Reading the index requires a couple of outdated dependencies that I wanted to avoid in the data-processing repository, as they have affected the "freshness" of dependencies across other plugins as well. The URL that you are pointing out is the REST API of "Easy Index", no longer the path pattern to the index files.